src/lib.rs

```diff
@@ -5,6 +5,8 @@ extern crate libc;
 extern crate uuid;
 
 pub mod error;
+pub mod errno;
```

Comment:
Minor suggestion: Might add some rustdoc above these modules so you're able to keep the various modules straight, esp. after not looking at this code for a while. I've been starting to use rustdoc where i can, and it really helps keep the organization straight + obviously serves as documentation. It looks like this

```rust
/**
 * Provides conversions from OS level error ints to meaningful strings (or whatever it actually does)
 */
pub mod errno;
```